src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,18 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  if (user) {
```

Comment:
The component should not return an empty string because it's not a valid React child. It would be better to return null or some default placeholder when the user data is not available.